src/tls/certificate.rs

```diff
@@ -227,6 +228,10 @@ impl WorkloadCertificate {
 
         let mut roots = RootCertStore::empty();
         roots.add_parsable_certificates(chain.iter().last().map(|c| c.der.clone()));
+        roots.add_parsable_certificates(vec![CertificateDer::from_pem_file(
```

Comment:
Stuff in this file is a temporary workaround for certificates b/c I don't really know how they work.